python/server/ServerWrapper.cpp

```diff
@@ -116,7 +119,7 @@ namespace {
         } else {
             Empire* empire = GetEmpire(empire_id);
             if (!empire) {
-                ErrorLogger() << "PythonServerWrapper::GenerateSitRep: couldn't get empire with ID " << empire_id;
+                ErrorLogger() << "ServerWrapper::GenerateSitRep: couldn't get empire with ID " << empire_id;
```

Comment:
Good idea, created PR #403. Modified your suggestion a bit, lets continue this discussion there.